charts/aws-ebs-csi-driver/templates/clusterrole-attacher.yaml

```diff
@@ -5,22 +5,22 @@ metadata:
   name: ebs-external-attacher-role
   labels:
     {{- include "aws-ebs-csi-driver.labels" . | nindent 4 }}
+# Do not modify the rules below manually, see `make update-sidecar-dependencies`
+# BEGIN AUTOGENERATED RULES
 rules:
-  - apiGroups: [ "" ]
-    resources: [ "persistentvolumes" ]
-    verbs: [ "get", "list", "watch", "update", "patch" ]
-  - apiGroups: [ "" ]
-    resources: [ "nodes" ]
```

Comment:
Calling out that we are losing the nodes permissions, is this because we haven't needed it in recent versions of the sidecar?

Reply:
See PR description